hashicorp/dev-portal
# updated certification images on homepage and landing

## 🔗 Relevant links

- [Preview link home page](https://dev-portal-git-tvupdate-certification-images-hashicorp.vercel.app/) 🔎
- [Preview link certifications](https://dev-portal-git-tvupdate-certification-images-hashicorp.vercel.app/certifications) 🔎
- [Asana task](https://app.asana.com/0/1202097197789424/1205161599037038/f) 🎟️

## 🗒️ What

Updated certification images on the home page `/` and certification landing `/certifications` 

## 🤷 Why

Terraform launched Associate 003, and that needed to be reflected in the images.

## 🛠️ How

Deleted the old images, named the new images the same as the old, and copied the new images in. 
Verified locally and checked with Design to confirm the new image size was intentional. 


## 🧪 Testing

- [ ] visit the home page 
  - ensure the image is loading and looks right
- [ ] visit `/certifications` 
  - ensure the hero is loading and looks right

## 💭 Anything else?

First PR 🎉 . Feedback welcome. But just make sure I am not about to break anything ;)
